### ISSUE TITLE
Why does `U.Nullable` adds `undefined` but not `null`?

### ISSUE THREAD

**Issue by thesunny** [pirix-gh/ts-toolbelt#113]
#### Describe the bug
The method `Union.Nullable` does not add the `null` type to the union

#### Reproduce the bug
```ts
type N = Union.Nullable<string>
// string | undefined
```

#### Expected behavior
```ts
type N = Union.Nullable<string>
// string | undefined | null
```

#### Possible Solution
```ts
export type Nullable<U extends Union> =
    U | undefined | null // added the | null
```

**Comment by pirix-gh** [comment]
Hi @thesunny, this is voluntary. The reason I did this is that most people (IMHO) see `null` as an anti-pattern, also called the "Billion dollar mistake", as it creates confusion in the codebase.

Well that was my only reason. Because in practice, TypeScript prevents this horrible mistake by not allowing `object` & `null` to overlap. So I built it this way, thinking that no one should use `null`.

Unfortunately, I don't think this should be changed, as the other `Nullable` utilities for `Object` and `List` behave the same way. I would like to keep stuff even. What we could do instead, if you really want this type here, we could put it into the `Community` module. But I think that it would be easier for you to just do `| null`, it's short.
